components/organisms/BlogSection/blog-section.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+
+import { DevToArticleResults } from './blog-section.types';
+import { BlogCard } from '@components/atoms';
+
+export function BlogSection() {
+    const [blogData, setBlogData] = useState<DevToArticleResults[] | []>([]);
+
+    useEffect(() => {
+        const getBlogs = async () => {
+            const response = await fetch(
+                'https://dev.to/api/articles?username=helen8297'
+            );
+            const data: DevToArticleResults[] = await response.json();
+            setBlogData(data);
+        };
+
+        getBlogs();
+    }, []);
+
+    console.log(blogData[1]);
+
```

Comment:
```suggestion
If(blogData.length <= 0) return null
```
Something like this might help with some of the type errors you are seeing.